### PR TITLE
This fixes #238

### LINKED ISSUE #238: Placeholder ISC23 web page needed
This is the URL we have slated in the call for submissions: https://io500.org/pages/bof-isc23

### PR DESCRIPTION
**Pull request by jeanbez**
Update CFS, submission page, and minor fixes to integrate with new submission system.

### CODE REVIEW

**Review by gmarkomanolis — APPROVED** [review]

File: templates/Pages/cfs_isc23.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    <p>
+        Once again, we encourage you to
+        <?php echo $this->Html->link('submit',
+            [ 'controller' => 'pages', 'action' => 'display', 'submission' ],
+            [ 'class' => 'link' ]);
+         ?>
+        to join our community, and to attend the  
+        <?php echo $this->Html->link('ISC23 BoF',
+            [ 'controller' => 'pages', 'action' => 'display', 'bof-sc22' ],
```
Comment (gflofst) [inline templates/Pages/cfs_isc23.php:97]: should be the ISC23 bof page

**Review by gflofst — CHANGES_REQUESTED** [review]
The bof page should be updated to isc23. Otherwise, looks good to me.

**Review by adilger — APPROVED** [review]

File: templates/Pages/cfs_isc23.php (hunk 1 above)
Comment (jeanbez) [reply templates/Pages/cfs_isc23.php:97]: @gflofst we do not have that page yet. I'll copy from last year, but let me know what changes need to be made.

Comment (gflofst) [reply templates/Pages/cfs_isc23.php:97]: I'm getting a server error at the moment for io500.org. Any ideas?

Comment (jeanbez) [reply templates/Pages/cfs_isc23.php:97]: @gflofst I just cleared the cache, it looks ok. Please let me know if you still run into issues.

Comment (gflofst) [reply templates/Pages/cfs_isc23.php:97]: @jeanbez website is back up. I'll get a placeholder page up for ISC23 and then we can get it live and linked in.

**Comment by adilger** [comment]
(Accidentally) replaced by PR #242.

**Comment by adilger** [comment]
The remaining changes are included in #245

**Pull request closed without merging**